install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -76,20 +68,14 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 					// TODO consider to provide it as a part of image label or rather inline ${ide} and ${supervisor} in index.html
 					// to decouple blobserve and code image
 					InlineStatic: []blobserve_config.InlineReplacement{{
+						Search:      "{{WORKBENCH_WEB_BASE_URL}}",
+						Replacement: "${ide}",
+					}, {
 						Search:      "window.location.origin;",
 						Replacement: "'${ide}';",
-					}, {
-						Search:      "${window.location.origin}",
-						Replacement: ".",
-					}, {
-						Search:      "value.startsWith(window.location.origin)",
-						Replacement: "value.startsWith(window.location.origin) || value.startsWith('${ide}')",
 					}, {
 						Search:      "./out",
 						Replacement: "${ide}/out",
-					}, {
-						Search:      "./node_modules",
-						Replacement: "${ide}/node_modules",
```

Comment:
These search values do not exist in `index.html` a long time ago so removing them it's fine